examples/server/server.cpp

```diff
@@ -2605,19 +2605,27 @@ static json format_detokenized_response(std::string content)
 
 static void log_server_request(const httplib::Request &req, const httplib::Response &res)
 {
-    LOG_INFO("request", {
-                            {"remote_addr", req.remote_addr},
-                            {"remote_port", req.remote_port},
-                            {"status", res.status},
-                            {"method", req.method},
-                            {"path", req.path},
-                            {"params", req.params},
-                        });
+    // skip GH copilot requests when using default port
+    if (req.path == "/v1/health" || req.path == "/v1/completions")
```

Comment:
Excellent! Thanks, it was flooding my logs.

Reply:
@ggerganov I read too fast, why ignoring `/completions` as it's the main purpose of this logger hook function ?

Reply:
Hm, I think `server` handles only the `/completion` and `/v1/chat/completions` endpoints:

https://github.com/ggerganov/llama.cpp/blob/34710525045ef9b4bec3c900ecdae11685f6862d/examples/server/server.cpp#L2952-L2954

https://github.com/ggerganov/llama.cpp/blob/34710525045ef9b4bec3c900ecdae11685f6862d/examples/server/server.cpp#L3051-L3053

It does not handle `/v1/completions`. This endpoint is what GH copilot hits and I thought to just filter it to avoid the spam from using it while `server` is running.

But we can re-enable this since just switching the server port to something different from 8080 also accomplishes the same thing

Reply:
Thanks for explaination, I double read wrongly :). Sure let's keep it.